.github/testing_frameworks.md

```diff
@@ -0,0 +1,21 @@
+## Testing frameworks
+
+Here are the frequently-used WiredTiger testing frameworks (broken down by testing types):
+
+- Unit testing
+  - [Catch2](https://github.com/wiredtiger/wiredtiger/tree/develop/test/unittest)
+- End-to-end testing
+  - [Python suite](https://github.com/wiredtiger/wiredtiger/tree/develop/test/suite)
+  - [C suite](https://github.com/wiredtiger/wiredtiger/tree/develop/test/csuite)
+  - [Model test](https://github.com/wiredtiger/wiredtiger/tree/develop/test/model)
+- Stress testing
+  - [format test](https://github.com/wiredtiger/wiredtiger/tree/develop/test/format)
+  - [checkpoint test](https://github.com/wiredtiger/wiredtiger/tree/develop/test/checkpoint)
+  - [CPP Suite](https://github.com/wiredtiger/wiredtiger/tree/develop/test/cppsuite)
+  - [testy] (https://github.com/wiredtiger/testy)
```

Comment:
`testy` displays differently to the other links, with the hyperlink appearing as follows, rather than `testy` being the text for the hyperlink:

`[testy] (https://github.com/wiredtiger/testy)`

This is because there is an unwanted space between `[testy] `and its url (unlike the other links)

Reply:
Good catch! Fixed in [519074e](https://github.com/wiredtiger/wiredtiger/pull/10973/commits/519074e193b20b8057409a292bc5e6643ceb1a69)